spack-ext/lib/jcsda-emc/spack-stack/stack/stack_env.py

```diff
@@ -100,7 +100,7 @@ def __init__(self, **kwargs):
 
```

Comment:
This is probably from the other open PR #1294 that needs to be merged first?

Reply:
Correct